src/os/FuseStore.cc

```diff
@@ -1127,7 +1195,9 @@ static int os_statfs(const char *path, struct statvfs *stbuf)
 static struct fuse_operations fs_oper = {
   getattr: os_getattr,
   readlink: 0,
+#if FUSE_VERSION < FUSE_MAKE_VERSION(3, 0)
   getdir: 0,
+#endif
```

Comment:
filled with 0 by default.   this is no needed

Reply:
Will fix it.

Reply:
@ukernel 
 
I am afraid we couldn't delete it due to strict check in makefile, please see

https://github.com/ceph/ceph/pull/34531#issuecomment-615821206